packages/fxa-settings/src/components/ButtonDownloadRecoveryKeyPDF/fonts/README.md

```diff
@@ -9,7 +9,7 @@ Noto is a collection of fonts with multiple weights and widths in sans, serif, m
 Read more: https://fonts.google.com/noto
 These fonts are licensed under the [Open Font License](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL).
 
-For most of the locales supported by Firefox accounts (see `supportedLanguages.json`), we can default to the standard Noto Sans, with a few exceptions that require specific fonts:
```

Comment:
The recovery key PDF has the Firefox logo in the top corner and will need to be updated to Mozilla. I can look into this next week since you're on PTO!

Reply:
That'd be great. Thanks!

Reply:
@dschom I've pushed a commit with recovery key PDF updates - can be tested from the "ButtonDownloadRecoveryKeyPDF" component in storybook. :)

Reply:
@vpomerleau I am validating that the PDF is indeed updated.